include/netutils/webclient.h

```diff
@@ -420,6 +462,20 @@ struct webclient_poll_info
   unsigned int flags; /* OR'ed WEBCLIENT_POLL_INFO_xxx flags */
 };
 
+struct webclient_conn_s
+{
+  bool tls;
+
+  /* for !tls */
+
+  int sockfd;
+  unsigned int flags;
+
+  /* for tls */
+
+  struct webclient_tls_connection *tls_conn;
```

Comment:
Good to add `FAR` to all the pointers in future

Reply:
sure